adapters/criteo/criteo_test.go

```diff
@@ -0,0 +1,425 @@
+package criteo
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"github.com/mxmCherry/openrtb/v14/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+
+	// Setup:
+	bidder, buildErr := BuilderWithGuidGenerator(
+		openrtb_ext.BidderCriteo,
+		config.Adapter{
+			Endpoint: "https://bidder.criteo.com/cdb?profileId=230",
+		},
+		newFakeGuidGenerator("00000000-0000-0000-00000000"),
+	)
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	// Execute & Verify:
+	adapterstest.RunJSONBidderTest(t, "criteotest", bidder)
+}
+
+func TestNewCriteoRequest(t *testing.T) {
```

Comment:
I looks like the tests other than `TestJsonSamples` are unit tests of the `models.go` code. If so, please move to `models_test.go` to make the correlation more explicit.

Reply:
Ack